src/main/java/space/obminyashka/items_exchange/model/EmailConfirmationCode.java

```diff
@@ -37,4 +37,9 @@ public EmailConfirmationCode(UUID id, User user, LocalDateTime expiryDate) {
         this.user = user;
         this.expiryDate = expiryDate;
     }
+
+    public EmailConfirmationCode(UUID id, int expirationHours) {
```

Comment:
A constructor with less amount of parameters is always placed above it with bigger ones.
Beside that from the current constructor you can invoke the second one with null as a user parameter.